source/class/qx/ui/table/pane/Scroller.js

```diff
@@ -1973,16 +1973,14 @@ qx.Class.define("qx.ui.table.pane.Scroller", {
     _onFocusinCellEditorAddBlurListener(e) {
       this.debug("executed FOCUSIN event listener for hash: " + e.getTarget().$$hash);
       qx.event.Timer.once(function() {
-        this._cellEditor.addListenerOnce('blur', this._onBlurCellEditorStopEditing, this);
-        this.debug('added BLUR listener to hash: ' + this._cellEditor.$$hash);
+        this._cellEditor.addListener('focusout', this._onFocusoutCellEditorStopEditing, this);
+        this.debug('added FOCUSOUT listener to hash: ' + this._cellEditor.$$hash);
```

Comment:
What does this do when the cell editor is a modal window instead of inline editor? Blur wouldn't happen with a modal window, but focusout could, and that shouldn't cause the modal window to close just because the mouse pointer moved out of that window...